### PR TITLE
DDP-7306. Decimal support (rebased)

### PR DESCRIPTION
**Pull request by Trimax**
Same changes, but rebased

### CODE REVIEW

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/instance/validation/DecimalRangeRule.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+/**
+ * A validation rule that checks numeric integer value is within an optional min/max range, inclusive.
+ */
+public class DecimalRangeRule extends Rule<DecimalAnswer> {
+
+    @SerializedName("min")
+    private BigDecimal min;
+
+    @SerializedName("max")
+    private BigDecimal max;
```
Comment (bskinner) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/instance/validation/DecimalRangeRule.java]: What does the JSON serialization of an object of this class look like?

**Review by bskinner — COMMENTED** [review]
Looks good- just a question about the JSON serialization of the `DecimalRangeRule`.

**Review by ssettipalli — DISMISSED** [review]
looks good to me

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/instance/validation/DecimalRangeRule.java (hunk 1 above)
Comment (Trimax) [reply pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/instance/validation/DecimalRangeRule.java]: Good point. This should be changed to the DecimalDef as well. Like for Answer

**Review by ssettipalli — APPROVED** [review]

**Pull request merged**